### ISSUE TITLE
Unable to connect to preset Telnet

### ISSUE THREAD

**Issue by fu-sen** [niruix/sshwifty#17]
I have added the following to `sshwifty.conf.json`, These are Japanese Telnet / ssh BBS:
```
  ：

  "Presets": [
    {
      "Title": "Fusen no heya (balloon.tel)",
      "Type": "Telnet",
      "Host": "balloon.tel:55123",
      "Meta": {
        "Encoding": "shift-jis"
      }
    },
    {
      "Title": "Fusen no heya (fusen.ddns.net)",
      "Type": "Telnet",
      "Host": "fusen.ddns.net:55123",
      "Meta": {
        "Encoding": "shift-jis"
      }
    },
    {
      "Title": "Seiwa net",
      "Type": "Telnet",
      "Host": "jp3tlc.no-ip.biz",
      "Meta": {
        "Encoding": "shift-jis"
      }
    },
    {
      "Title": "Virus net",
      "Type": "Telnet",
      "Host": "211.133.151.119",
      "Meta": {
        "Encoding": "shift-jis"
      }
    },
    {
      "Title": "WhiteWing",
      "Type": "Telnet",
      "Host": "bbs.whitewing.gr.jp",
      "Meta": {
        "Encoding": "shift-jis"
      }
    },
    {
      "Title": "Communitex",
      "Type": "Telnet",
      "Host": "www.maruo.co.jp",
      "Meta": {
        "Encoding": "shift-jis"
      }
    },
    {
      "Title": "Hanzou BBS (Telnet)",
      "Type": "Telnet",
      "Host": "ktbbs.hanzou.jp:11123",
      "Meta": {
        "Encoding": "shift-jis"
      }
    },
    {
      "Title": "Hanzou BBS (ssh)",
      "Type": "Telnet",
      "Host": "ktbbs.hanzou.jp:11111",
      "Meta": {
        "Encoding": "shift-jis",
        "User": "bbs",
        "Authentication": "bbs"
      }
    },
    {
      "Title": "FENIX (shift-jis)",
      "Type": "Telnet",
      "Host": "akari.fenix.ne.jp",
      "Meta": {
        "Encoding": "shift-jis"
      }
    },
    {
      "Title": "Wappy nets (wapiko.ddo.jp)",
      "Type": "Telnet",
      "Host": "wapiko.ddo.jp",
      "Meta": {
        "Encoding": "shift-jis"
      }
    },
    {
      "Title": "Wappy nets (wapiko.jp)",
      "Type": "Telnet",
      "Host": "wapiko.jp",
      "Meta": {
        "Encoding": "shift-jis"
      }
    }
  ],
  "OnlyAllowPresetRemotes": true
```
As a result, I can only connect my own BBS (Fusen no heya, balloon.tel = fusen.ddns.net),
which seems to connect only locally.

**Comment by fu-sen** [comment]
Do you get it. Telnet required a port number to be specified.
```
{
      "Title": "Seiwa net",
      "Type": "Telnet",
      "Host": "jp3tlc.no-ip.biz:23",
      "Meta": {
        "Encoding": "shift-jis"
      }
    },
```
I don't think this is a bug, but you may need to consider Telnet's default port numbers.

**Comment by niruix** [comment]
Nice catch. I'll figure something out.

Thank you for reporting.

**Comment by niruix** [comment]
Hello, sorry for the long delay, I been busy on something else lately. 

I've published a newer version `0.2.1-beta-release` which will automatically fill up the port number for Presets with a default value when the port number is not provided. This change will fix the issue above.

I'll close this issue for now, please reopen it if the fix did not resolve the problem. 

Thank you!
